src/nti/transactions/transactions.py

```diff
@@ -237,6 +253,16 @@ def do(*args, **kwargs):
     transaction.get().join(result)
     return result
 
+def do_last(*args, **kwargs):
+    """
+    Establishes a IDataManager in the current transaction that will attempt to
+    execute *after* all other DataManagers have had their say.
+    See :class:`ObjectDataManager` for the possible arguments.
+    """
+    result = ExecuteLastObjectDataManager(*args, **kwargs)
```

Comment:
Rather than reimplementing this, what about something like this:

```python
def do(*args, **kwargs):
    klass = kwargs.pop('datamanager_class', ObjectDataManager) # new argument
    result = klass(*args, **kwargs)
    ... # as before

def do_XXX(*args, **kwargs):
   kwargs['datamanager_class'] = XXXDataManager
   return do(*args, **kwargs)
```